<documentation title="Opening Brace in Control Structures">
    <standard>
    <![CDATA[
    Function declarations follow the "BSD/Allman style". The opening brace is on the line following the control structure and is indented to the same column as the start of the control structure.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: brace on next line">
        <![CDATA[
foreach($array as $key => $val)
<em>{</em>
    ...
}
        ]]>
        </code>
        <code title="Invalid: brace on same line">
        <![CDATA[
foreach($array as $key => $val) <em>{</em>
    ...
}
        ]]>
        </code>
    </code_comparison>
</documentation>
